src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, 
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+      src="./images/imac.jpeg" 
+      class="card__img" 
+      alt="APPLE A1419 iMac 27"
+      >
+      <h2 class="card__title">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__rating">
+        <div class="block-stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+            
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+      <div class="card__cost">
+        <p class="card__price">Price:</p>
+        <p class="card__amount">$2,199</p>
+      </div>
+      <div class="card__button">
+        <a 
+        class="card__link" 
+        href="#Buy" 
+        data-qa="hover"
+        >
+        Buy
+        </a>
+      </div>
```

Comment:
We can add a link without wrap container
```suggestion
      <a 
        class="card__button" 
        href="#Buy" 
        data-qa="hover"
      >
        Buy
      </a>
```
and join styles (card__button and card__link)